modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/PostalAddressResourceTest.java

```diff
@@ -121,6 +199,7 @@ protected PostalAddress randomPostalAddress() {
 				postalCode = RandomTestUtil.randomString();
 				primary = false;
 				streetAddressLine1 = RandomTestUtil.randomString();
+				subtype = _subtype1.getKey();
```

Comment:
leave this empty so all the tests use an empty subtype. Then please add also a _testPostPostalAddress with a subtype and then we have the patch that checks that we also change it